tests/test_reader.py

```diff
@@ -34,7 +34,7 @@ def test_list_signals_exclude_raw():
     info = signals[0]
     assert info.name == "abm/calib_shot"
 
-
+@pytest.mark.skip(reason="Uses MAST Client, not available on CSD3")
```

Comment:
Don't think we should mentioned CSD3. Just say that the pyuda client is not available.

Reply:
done :)